r/tests/testthat/test-compute-sort.R

```diff
@@ -15,9 +15,7 @@
 # specific language governing permissions and limitations
 # under the License.
 
-context("compute: sorting")
-
-library(dplyr)
+suppressPackageStartupMessages(library(dplyr))
```

Comment:
This might be a little bit odd, but should we create a helper that does this `quiet_library()` so it's a bit less to type?

Reply:
A better way for all of these except for bit64 is probably `library(warn.conflicts = FALSE)`